src/Destructurama.JsonNet/Destructurama.JsonNet.csproj

```diff
@@ -1,15 +1,15 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net452;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>net462;netstandard2.0</TargetFrameworks>
```

Comment:
@nblumhardt Keep net462 or drop?

Reply:
I think there's value in 462, I'd also consider adding net6.0; no strong feelings though

Reply:
I'll publish `net462;netstandard2.0` in v3. We can add net6 later if some issue arises.